Xplat/src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -188,6 +188,7 @@
   "tag.botania.mini_flowers": "Petite Flowers",
   "tag.botania.mana_using_items": "Mana Using Items",
   "tag.botania.lens": "Mana Lenses",
+  "tag.botania.lens_glue": "Lens Glues",
```

Comment:
I feel like either, the tag should be named `lens_glues`, or the translation should be `"Lens Glue"`. Not sure which i prefer. I think the convention for tags is to be plural, though.

Reply:
I was going off "lens", which has the internal name be singular and the display name be plural, but I could change the internal name to be plural as well, if desired.